config/dynamic.js

```diff
@@ -87,6 +87,8 @@ export const app = {
 	sentryURI: process.env.SENTRY_URI || 'https://7ce141b23c4a4e6091c206d08442f0e9@o7540.ingest.sentry.io/1201287',
 	sentryTraceSampleRate: process.env.SENTRY_TRACE_RATE || 0.25,
 	snowplowUri: process.env.SNOWPLOW_URI || 'events.fivetran.com/snowplow/v5qt54ocr2nm',
+	stellateGraphqlUri: process.env.STELLATE_GRAPHQL_URI || null,
```

Comment:
Which config is `dynamic`?

Reply:
This is the file we now use for all real environments.